org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -394,6 +399,10 @@ private synchronized void doNodeInitialisation() {
                     stateDescriptions.put(channel.getUID(), stateDescription);
                 }
             }
+
+            if (initializeDevice) {
```

Comment:
Why is there the need for the ```if``` statement here? For the sake of simply writing the property on every startup (not a high frequency operation) you could remove this and just update the property?

Reply:
Will remove the `if` statement